scripts/generate_hub_stats.py

```
@@ -0,0 +1,142 @@
+BUCKET = "dashboard-analytics"
+HUB_KEY = "hub-stats/hub_stats.json"
+BWS_KEY = "hub-stats/hub_stats_bws.json"
+
+
+def es_query(query: dict) -> dict:
+    url = f"http://{ES_HOST}:{ES_PORT}/dpla_alias/_search"
+    data = json.dumps(query).encode()
+    req = urllib.request.Request(
+        url, data=data, headers={"Content-Type": "application/json"}
+    )
+    with urllib.request.urlopen(req, timeout=30) as resp:
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)